### PR TITLE
11695 tor: update to version 0.4.1.5

### PR DESCRIPTION
**Pull request by svschmel**
My first PR :)
i have updated my tor-server to Version 0.4.1.5
=> https://metrics.torproject.org/rs.html#details/AD0FCFD83EA4899DB34735E599701592DEA13690

### CODE REVIEW

File: components/network/tor/Makefile [hunk 1]
```diff
@@ -72,3 +72,13 @@ REQUIRED_PACKAGES += SUNWcs
 REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += system/library/math
+# Auto-generated dependencies
```
Comment (Mno-hime) [inline components/network/tor/Makefile:75]: `REQUIRED_PACKAGES` above the generated section are duplicate to those in the generated section.

File: components/network/tor/tor.p5m [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 #
 # A full copy of the text of the CDDL should have accompanied this
-# source. A copy of the CDDL is also available via the Internet at
+# source.  A copy of the CDDL is also available via the Internet at
 # http://www.illumos.org/license/CDDL.
 #
 
 #
 # Copyright 2014 Alexander Pyhalov. All rights reserved.
+# Copyright 2018 Sven Schmeling
```
Comment (Mno-hime) [inline components/network/tor/tor.p5m:14]: 2019?

File: components/network/tor/tor.p5m [hunk 3]
```diff
@@ -33,13 +35,20 @@ dir path=var/log/tor group=daemon mode=0755 owner=daemon
 file etc/tor/torrc.sample path=etc/tor/torrc mode=0644 preserve=true
 
 file usr/bin/tor path=usr/lib/tor mode=0555
+
```
Comment (Mno-hime) [inline components/network/tor/tor.p5m:38]: This blank space is not required.

File: components/network/tor/tor.p5m [hunk 4]
```diff
@@ -33,13 +35,20 @@ dir path=var/log/tor group=daemon mode=0755 owner=daemon
 file etc/tor/torrc.sample path=etc/tor/torrc mode=0644 preserve=true
 
 file usr/bin/tor path=usr/lib/tor mode=0555
```
Comment (Mno-hime) [inline components/network/tor/tor.p5m:37]: Is there a reason to keep the `tor` binary in `usr/lib/`? If you decide to change it, make sure `files/tor.sh` is changed accordingly.

**Review by Mno-hime — COMMENTED** [review]
Excellent PR! Thanks for updating Tor, it's a shame we have such an old version. There are few things to polish in the PR, otherwise looks good.

File: components/network/tor/Makefile (hunk 1 above)
Comment (svschmel) [reply components/network/tor/Makefile:75]: Yes, that's right,
In my new version i fixed it.

File: components/network/tor/tor.p5m (hunk 2 above)
Comment (svschmel) [reply components/network/tor/tor.p5m:14]: Yes, of course :)

File: components/network/tor/tor.p5m (hunk 3 above)
Comment (svschmel) [reply components/network/tor/tor.p5m:38]: fixed in the version

File: components/network/tor/tor.p5m (hunk 4 above)
Comment (svschmel) [reply components/network/tor/tor.p5m:37]: i used the p5m-file from the old version as a template.
In the past many programs were installed in the lib-directory :)
In my new version the tor-binary is in the (/usr/)bin-dir. I changed the tor.sh script for the new position.
At the moment i am testing this new package.

Comment (Mno-hime) [reply components/network/tor/tor.p5m:37]: @svschmel Cool, I'll review once you push those changes.

**Comment by Mno-hime** [comment]
Closing in favor of https://github.com/OpenIndiana/oi-userland/pull/5317.

**Pull request closed without merging**